### PR TITLE
feat: Onboard New Fec dataset

### PR DESCRIPTION
**Pull request by gkodukula**
## Description
Dataset: FEC

- FEC Committee Contributions
- 2016 FEC Candidate Committee
- 2016 FEC Candidate
- 2016 FEC Opex
- 2016 Other Committee Tx
- 2016 FEC Committee
- 2016 FEC Individuals
- 2018 FEC Committee
- 2018 FEC Opex
- 2018 FEC Committee Contributions
- 2018 FEC Candidate Committee
- 2018 FEC Other Committee Tx
- 2018 FEC Candidate
- 2020 FEC Candidate Committee
- 2020 FEC Candidate
- 2020 FEC Other Committee Tx
- 2020 FEC Committee
- 2020 FEC Committee Contributions
- 2020 FEC Opex
- 2020 FEC Individuals - Step 3: Ingest

## Checklist

- [x] **(Required)** This pull request is appropriately labeled
- [x] Please merge this pull request after it's approved

Use the sections below based on what's applicable to your PR and delete the rest:

### Feature
- [x] I'm adding or editing a feature
- [x] I have updated the [`README`](https://github.com/GoogleCloudPlatform/public-datasets-pipelines/blob/main/README.md) accordingly
- [x] I have added/revised tests for the feature

### Data Onboarding
- [x] I'm adding or editing a dataset
- [x] The [Google Cloud Datasets team](mailto:cloud-datasets-onboarding@google.com) is aware of the proposed dataset
- [x] I put all my code inside  `datasets/fec` and nothing outside of that directory

### CODE REVIEW

**Comment by nlarge-google** [comment]
Gowtham,  there are multiple issues with the transform_csv python file:

**Review by nlarge-google — CHANGES_REQUESTED** [review]
As specified in my comment

File: datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 1]
```diff
@@ -49,8 +49,10 @@ def main(
```
Comment (nlarge-google) [inline datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py:49]: Remove all blank lines within functions

File: datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 2]
```diff
@@ -49,8 +49,10 @@ def main(
 
     logging.info("Creating 'files' folder")
     pathlib.Path("./files").mkdir(parents=True, exist_ok=True)
-
-    if "individuals" not in pipeline_name:
+    if (
+        "individuals" not in pipeline_name
+        and "other_committee_tx_2020" not in pipeline_name
+    ):
         logging.info(f"Downloading file from {source_url}...")
```
Comment (nlarge-google) [inline datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py:56]: Move logging to respective function definition.

File: datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py [hunk 3]
```diff
@@ -80,19 +91,19 @@ def main(
...
+            df.columns = csv_headers
             df["transaction_dt"] = df["transaction_dt"].astype(str)
             date_for_length(df, "transaction_dt")
             df = resolve_date_format(df, "transaction_dt", pipeline_name)
-
         else:
+            df.columns = csv_headers
             pass
 
         logging.info(f"Saving to output file.. {target_file}")
```
Comment (nlarge-google) [inline datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py:109]: Move logging to respective function definition.

**Review by nlarge-google — CHANGES_REQUESTED** [review]

File: datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 3 above)
Comment (gkodukula) [reply datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py]: done

File: datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 2 above)
Comment (gkodukula) [reply datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py]: done

File: datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py (hunk 1 above)
Comment (gkodukula) [reply datasets/fec/pipelines/_images/run_csv_transform_kub/csv_transform.py]: done

**Review by nlarge-google — APPROVED** [review]
Thanks Gowtham.

**Comment by nlarge-google** [comment]
@gkodukula This is now merged and deployed to production.  Please ensure execution cycle completes successfully.

**Pull request merged**